models/DataObject/ClassDefinition/Data/QuantityValue.php

```diff
@@ -708,4 +708,17 @@ public function getFilterCondition($value, $operator, $params = [])
 
         return implode(' OR ', $conditions);
     }
+
+    public function isEqual($oldValue, $newValue)
+    {
+        if(!$oldValue instanceof Model\DataObject\Data\QuantityValue) {
```

Comment:
can the new value be null?

Reply:
yes, but I think what is missing is that the old and new value can be null, therefore it is still equal

Reply:
? I do not understand the question. Of course it can as this is a public method - old and new value can be of any type. Or what do you mean?

Reply:
Added a check for old and new value being `null`